### PR TITLE
UIEH-1163: Replace loadMoreButton to PrevNextButtons

### PR DESCRIPTION
**Pull request by Vladyslav-Velytskyi**
# UIEH-1163 View Packages Record > List of Titles Accordion | Replace Load More AND Apply Next/Previous Pagination component

## Purpose
- Create `<PrevNextComponent />`
- Add tests
- Update reducer for `packageTitles`

Issue: [UIEH-1163](https://issues.folio.org/browse/UIEH-1163)

#### TODOS and Open Questions
After all replacement `<ScrollView />` component should be updated

## Screenshots
Pb6954D7mJ

### CODE REVIEW

**Review by mariiaschv — APPROVED** [review]

File: src/components/prev-next-buttons/prev-next-buttons.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const PrevNextButtons = ({
+  page,
+  setPage,
+  totalResults,
+  isLoading,
+}) => {
+  const intl = useIntl();
+
+  const maxItemsLength = page * PAGE_SIZE;
+  const disabledNext = maxItemsLength > totalResults;
```
Comment (BogdanDenis) [inline src/components/prev-next-buttons/prev-next-buttons.js:22]: Should this be `>=`? If we have equal total results and currently displayed results I think we should disable the button also

File: src/components/prev-next-buttons/prev-next-buttons.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    ? totalResults
+    : maxItemsLength;
+
+  return (
+    <div className={styles['button-wrapper']}>
+      <div className={styles['button-container']}>
+        <Button
+          disabled={page === 1 || isLoading}
+          buttonStyle="none"
+          data-testid='previous-button'
```
Comment (BogdanDenis) [inline src/components/prev-next-buttons/prev-next-buttons.js:34]: Please use double quotes

File: src/components/prev-next-buttons/prev-next-buttons.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          onClick={() => {
+            setPage(page - 1);
+          }}
+        >
+          <Icon size="small" icon="caret-left">
+            <span>{intl.formatMessage({ id: 'ui-eholdings.previous' })}</span>
```
Comment (BogdanDenis) [inline src/components/prev-next-buttons/prev-next-buttons.js:40]: Do we need an additional `span` here?

File: src/components/prev-next-buttons/prev-next-buttons.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          </Icon>
+        </Button>
+        <div className={styles['prev-next-page-info']}>
+          <div>{dataStartIndex}</div>&nbsp;-&nbsp;<div>{dataEndIndex}</div>
+        </div>
+        <Button
+          disabled={disabledNext || isLoading}
+          buttonStyle="none"
+          data-testid='next-button'
```
Comment (BogdanDenis) [inline src/components/prev-next-buttons/prev-next-buttons.js:49]: Single quotes here also

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

**Pull request merged**